plugins/modules/cloudformation.py

```diff
@@ -472,6 +476,9 @@ def update_stack(module, stack_params, cfn, events_limit):
     if 'TemplateBody' not in stack_params and 'TemplateURL' not in stack_params:
         stack_params['UsePreviousTemplate'] = True
 
+    if module.params['stack_policy_on_update_body'] is not None:
+         stack_params['StackPolicyDuringUpdateBody'] = module.params['stack_policy_on_update_body']
```

Comment:
```suggestion
    if module.params['stack_policy_on_update_body'] is not None:
        stack_params['StackPolicyDuringUpdateBody'] = module.params['stack_policy_on_update_body']
```

There's a Sanity test failure you've got an extra space in here (total of 9 instead of 8)